packages/plugins/documentation/server/services/helpers/utils/loop-content-type-names.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const _ = require('lodash');
+
+/**
+ * @description A reusable loop for building api endpoint paths and component schemas
+ *
+ * @param {object} api - Api information to pass to the callback
+ * @param {function} callback - Logic to execute for the given api
+ *
+ * @returns {function}
+ */
+const loopContentTypeNames = (api, callback) => {
+  let result = {};
+  for (const contentTypeName of api.ctNames) {
+    // Get the attributes found on the api's contentType
+    const uid = `${api.getter}::${api.name}.${contentTypeName}`;
+    const { attributes, info: contentTypeInfo } = strapi.contentType(uid);
+
+    // Get the routes for the current api
+    const routeInfo =
+      api.getter === 'plugin'
+        ? strapi.plugin(api.name).routes['content-api']
+        : strapi.api[api.name].routes[contentTypeName];
+
+    // Return early when routeInfo is undefined
+    if (!routeInfo) return;
+
+    // Uppercase the first letter of the api name
+    const apiName = _.startCase(api.name);
```

Comment:
startCase will no just upper the first later bu change a bunch of other things. I reco using upperFirst in place of startCase